app/sbna-poc/v1_0__initial-form/fields/accommodation.ts

```diff
@@ -644,7 +647,8 @@ export const noAccommodationFields: Array<FormWizard.Field> = [
     text: 'Why does [subject] have no accommodation?',
     hint: { html: noAccommodationHint, kind: 'html' },
     code: 'no_accommodation_reason',
-    type: FieldType.Radio,
+    type: FieldType.CheckBox,
+    multiple: true,
     validate: [{ type: ValidationType.Required, message: 'Select why they have no accommodation' }],
     options: [
       { text: 'Alcohol related problems', value: 'ALCOHOL_PROBLEMS', kind: 'option' },
```

Comment:
I can't see on github but was this also changed, on line 658?
from:
labelClasses: getMediumLabelClassFor(FieldType.Radio), 
To:
labelClasses: getMediumLabelClassFor(FieldType.CheckBox)

Reply:
well spotted! have updated it now, thanks

Reply:
great :)